packages/core/src/lib/icon/icon.svelte

```diff
@@ -0,0 +1,69 @@
+<!--
+@component
+  
+A component that renders SVG icons from the @mdi/js package
+```svelte
+  <Icon
+    name='camera-outline'
+    size='xl' 
+  />
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import cx from 'classnames';
+import { paths } from './icons';
+
+/**
```

Comment:
Non-exported variables do not need a docstring unless it makes the code more readable. I think the docstring is unnecessary in the case of Size, hasNameProperty, and sizes, but it's ultimately up to you.